### ISSUE TITLE
Discord server shut down?

### ISSUE THREAD

**Issue by Mylloon** [sudachi-emu/sudachi#51]

**Comment by jarrodnorwell** [comment]
Yes, Discord shut 3 of my servers down, 2 for emulation and 1 for friends and family as well as disabled my account. I'm currently waiting for a response from them.

**Comment by cgfrancisco** [comment]
My best bet bro is for you to have maybe at least a telegram as a backup social group for now the discord appeal is kinda 50/50 especially if where dealing with Uncle N*nt*ndo..

**Comment by ryzendew** [comment]
> Yes, Discord shut 3 of my servers down, 2 for emulation and 1 for friends and family as well as disabled my account. I'm currently waiting for a response from them.

https://github.com/revoltchat it has a flatpak and easy to use for the end user also self hosted by whoever makes it so it's decentralized and has the same features as discord mostly

**Comment by dtantono** [comment]
I heard Guilded or Matrix also good as well

**Comment by ryzendew** [comment]
> I heard Guilded or Matrix also good as well

Matrix is not good for end users

**Comment by BlazingFlames6073** [comment]
I came here to recommend matrixx as well but revolt might honestly be a better option for most users. I forgot about that project.

I'd also like to recommend lemmy(kbin is also an option) and mastodon if you want to make a complete move from other social media that might not be safe in the future for sudachi.

**Comment by mateomaui** [comment]
> I'd also like to recommend lemmy(kbin is also an option) 

Unless changes have happened, I would recommend against kbin, as it did have federation issues with propagating various actions onto lemmy, and just go with lemmy, if this suggestion is taken.

**Comment by BlazingFlames6073** [comment]
> > I'd also like to recommend lemmy(kbin is also an option)
> 
> Unless changes have happened, I would recommend against kbin, as it did have federation issues with propagating various actions onto lemmy (at least I think that was the problem), and just go with lemmy, if this suggestion is taken.

I agree. I wasn't sure if I should mention kbin but I did anyway since it's pretty well known to people who use lemmy. It definitely does have issues when you try looking at it from lemmy's side and kbin is overall less active than lemmy and doesn't have as many apps for users to choose so I do think that lemmy is a better option if this suggestion is ever taken. It's been sometime since I checked both services.

**Comment by mateomaui** [comment]
> > > I'd also like to recommend lemmy(kbin is also an option)
> > 
> > 
> > Unless changes have happened, I would recommend against kbin, as it did have federation issues with propagating various actions onto lemmy (at least I think that was the problem), and just go with lemmy, if this suggestion is taken.
> 
> I agree. I wasn't sure if I should mention kbin but I did anyway since it's pretty well known to people who use lemmy. It definitely does have issues when you try looking at it from lemmy's side and kbin is overall less active than lemmy and doesn't have as many apps for users to choose so I do think that lemmy is a better option if this suggestion is ever taken. It's been sometime since I checked both services.

I forgot about the apps situation, but that alone is worth going to lemmy. Just get voyagerapp with all the features for free, or use the web app version, and don't look back.
